gulp/hygiene.js

```diff
@@ -17,8 +17,8 @@ module.exports = (blueprint, gulp, plugins) => {
         return del(cleanDirs, { force: true });
     });
 
-    gulp.task("tslint", () => (
-        gulp.src(["*.js", "gulp/**/*.js", "packages/*/*.js"])
+    gulp.task("tslint-gulp", () => (
```

Comment:
ok so: the previous iteration of this task **ran tslint on all `.js` files** (hence the `package/**/*.js` glob).

this new iteration **runs tslint on the gulp code only** because JS files in packages are now linted by `tslint-<package>`.